packages/fhir-converter/src/templates/cda/DataType/Address.hbs

```diff
@@ -9,4 +9,5 @@
     "state": "{{addr.state._}}",
     "country": "{{addr.country._}}",
     "postalCode": "{{addr.postalCode._}}",
+    "period": {{>DataType/Period.hbs period=addr.useablePeriod}}
```

Comment:
tells you if the address is up to date by looking if its still in use. valuable to have